src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,67 @@
 package core.basesyntax;
 
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
```

Comment:
Use LocalDate instead of Date. Here is some background why you should use it primarily.

Reply:
Fix comment